server/src/main/java/io/unitycatalog/server/UnityCatalogServer.java

```diff
@@ -93,7 +93,22 @@ private void addServices(ServerBuilder sb) {
   public static void main(String[] args) {
     int port = 8080;
     if (args.length > 0) {
-      port = Integer.parseInt(args[0]);
+      if ("-p".equals(args[0]) || "--port".equals(args[0])) {
+        if (args.length > 1) {
+          try {
+            port = Integer.parseInt(args[1]);
+          } catch (NumberFormatException e) {
+            System.err.println("Invalid port number: " + args[1]);
+            return;
+          }
+        } else {
+          System.err.println("No port number provided after " + args[0]);
+          return;
+        }
+      } else {
+        System.err.println("Invalid argument: " + args[0] + ". Please use -p or --port to specify the port number.");
+        return;
+      }
```

Comment:
I think we can make this a bit more sophisticated by using a cli library.

Reply:
Sure, we can use the same cli library as examples/cli - I will update this.

Reply:
Sorry - just saw your message. I just pushed a commit with the commons-cli changes. Please let me know if this is fine and yeah meanwhile I'll check out the recommended one as well.

Reply:
One question: Should we use a different library just for this part of the code? Because in examples/cli each file uses commons-cli. So, using commons-cli for consistency with this module would make sense. Thanks!

Reply:
Let's keep it consistent and use commons-cli. If we transition to something else then we can do this here as well.